### DISCUSSION TITLE
incubator-streampark/streampark-spark/streampark-spark-cli/default.properties 注释乱码

### DISCUSSION THREAD

**Discussion by MakeUpForeverForMe** [apache/streampark#1646] [General]
incubator-streampark/streampark-spark/streampark-spark-cli/default.properties 注释乱码

#\u5FC5\u987B\u8BBE\u7F6E,\u6267\u884Cclass\u7684\u5168\u5305\u540D\u79F0

[链接](https://github.com/apache/incubator-streampark/blob/dev/streampark-spark/streampark-spark-cli/default.properties)

**Comment by youyangkou** [comment]
This is because the default properties is Unicode encoding, open IDEA in the setting can be set
![image](https://user-images.githubusercontent.com/33364356/194699317-0d1dd6a9-ddd7-4ec0-9a3a-5ac2a61bafde.png)
